src/module/treasure.js

```diff
@@ -1,32 +1,41 @@
 export const augmentTable = (table, html, data) => {
   // Treasure Toggle
+  const isTreasureTable = table.object.getFlag("ose", "treasure");
+
+  let treasureTableToggle = $("<div class='toggle-treasure' title='Toggle Treasure Table'></div>");
+  treasureTableToggle.toggleClass("active", isTreasureTable);
```

Comment:
[.toggleClass( className, state )](https://api.jquery.com/toggleClass/#toggleClass-className-state)

    className
    Type: [String](http://api.jquery.com/Types/#String)
    One or more classes (separated by spaces) to be toggled for each element in the matched set.
    state
    Type: [Boolean](http://api.jquery.com/Types/#Boolean)
    A boolean (not just truthy/falsy) value to determine whether the class should be added or removed.

While we're giving undefined for the state, it appears as though toggleClass treats it as true with no error returned. This results in unexpected behavior: it appears to default to toggled as active even though there's no flag by default.

Reply:
I understand, I intended to be a bit smarter than a ternary with 2 raw strings, and take advantage of JQuery for clarity. I put my questions regarding the handling of the `active` var above.